custom_model_runner/datarobot_drum/drum/model_adapter.py

```diff
@@ -265,11 +267,68 @@ def predict(self, data, model=None, **kwargs):
 
         return predictions
 
+    def try_to_fit_on_custom_wrapper(self, X, y, output_dir):
+        """
+        A user can surround an sklearn pipeline or estimator with the custom() function, importable
+        from drum, which will tag the object that is passed in with a to_upload variable.
+        This function searches thru all the pipelines and estimators imported from all the modules
+        in the code directory, and looks for this to_upload variable. If it finds it, it will
+        load the object here, and call fit on it. Then, it will serialize the fit model out
+        to the output directory. If it can't find the wrapper, it will return False, if it
+        successfully runs fit, it will return True, otherwise it will throw a DrumCommonException.
+
+        Returns
+        -------
+        Boolean, whether fit was run
+        """
+        model_dir_limit = 100
+        wrapped_object = None
+        sys.path.insert(0, self._model_dir)
+        files_in_model_dir = glob.glob(self._model_dir + "/*.py")
+        if len(files_in_model_dir) == 0:
+            return False
+        if len(files_in_model_dir) > model_dir_limit:
+            self._logger.warning(
+                "There are more than {} files in this directory".format(model_dir_limit)
+            )
+            return False
+        for filepath in files_in_model_dir:
+            filename = os.path.basename(filepath)
+            try:
+                module = __import__(filename[:-3])
+            except ImportError as e:
+                self._logger.warning(
+                    "File at path {} could not be imported: {}".format(filepath, str(e))
+                )
+                continue
+            for object_name in dir(module):
+                _object = getattr(module, object_name)
+                if isinstance(_object, sklearn.base.BaseEstimator):
+                    if hasattr(_object, "is_custom"):
```

Comment:
If we use `hasattr(_object, "is_custom"):` can we do it symmetrical when we set it:
Not use:  `estimator.is_custom = True`, but `estimator(object, "is_custom", True)`.
And then define `is_custom` as a constant and choose safer name, like `__drum_is_custom__`

Reply:
sure